nifi-nar-bundles/nifi-avro-bundle/nifi-avro-processors/src/main/java/org/apache/nifi/processors/avro/ConvertAvroToJSON.java

```diff
@@ -92,6 +103,7 @@
         .build();
 
     private List<PropertyDescriptor> properties;
+    private Schema schema = null;
```

Comment:
This could be marked by multiple different threads concurrently, so we need to make sure that it is protected. Marking it as 'volatile' should take care of this.